src/forms/handle_forms.ts

```diff
@@ -35,24 +37,53 @@ export class BaseForm {
     };
 }
 
-// CreateTicketForm handles creation and submission for creating a ticket from a post
+// OpenCreateTicketForm opens a new create ticket form
+export class OpenCreateTicketForm extends BaseForm {
+    handleSubmit = async (): Promise<AppCallResponse> => {
+        return {
+            type: AppCallTypes.FORM,
+            form: await newCreateTicketForm(this.call),
+        };
+    }
+}
+
+// CreateTicketForm updates the create ticket form with new values or creates
```

Comment:
then why not call it CreateOrUpdateTicketForm?

Reply:
That sort of makes sense, but my thought is that a form is a noun, not a verb.  In this case, I will only open a new `Create ticket form`.  The actions that can be performed on the form are updates to form values and submission of the form.

@mickmister thoughts on naming convention?

Reply:
then CreateOrUpdateCreateTicketForm - I know it doesn't sound very well, but at least it is unambiguous (to me)

Reply:
@catalintomai, this is actually a really good idea. please see [`08090e4` (#21)](https://github.com/mattermost/mattermost-app-zendesk/pull/21/commits/08090e4e96257fdead0902e48379c08ef018d73d)

I've renamed call functions, files, route paths, class names to make each very intentional.  I think it's important that we get this defined very well in this app since it will likely be used as an example for future JS apps.